docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_9.md

```diff
@@ -0,0 +1,86 @@
+    - <img src="./rna-seq-images/rna-seq-9-14.png" width="400"/>
+- The first two commands install dependencies needed for the data transfer. Click within each of these paragraphs (blocks of code) and then run each of them by clicking the `Play` button at the top of the window.
+    - <img src="./rna-seq-images/rna-seq-9-15.png" width="400"/>
+- The next paragraph will ask for your AnVIL API Key. Run this paragraph and a text box will apear. Open the `credentials.json` file on your computer and copy the api_key (without quotation marks) and paste it into the text box. Hit Enter on your keyboard to confirm.
+    - <img src="./rna-seq-images/rna-seq-9-16.png" width="400"/>
+- The next two paragraphs ask for credentials from the Cancer Genomics Cloud and BioData Catalyst. We may skip these paragraphs because we are not using these platforms.
+- The next paragraph asks for your CAVATICA Authentication Token. You can access this token by navigating to [https://cavatica.sbgenomics.com/developer/token](https://cavatica.sbgenomics.com/developer/token). Copy it to your clipboard.
+    - <img src="./rna-seq-images/rna-seq-9-17.png" width="400"/>
+- Back in the Data Cruncher, run this paragraph and paste your CAVATICA Authentication Token in the box. Hit Enter on your keyboard to confirm.
+    - <img src="./rna-seq-images/rna-seq-9-18.png" width="400"/>
+- The next paragraph includes a number of commands necessary for the transfer. Because we will only be using the AnVIL and CAVATICA tokens, we should remove the additional lines which set the `cgc_token` and `bdc_token`s. Delete these two lines from the paragraph before running it using the `Play` button above. The text in this box should read...
+
+```
+interop_client = DrsClient(
+    api_key=anvil_api_key,
+    cavatica_token=cavatica_token
+)
+```
+- Upload the PFB file into the Data Cruncher. This is the manifest of GTEx files downloaded from the AnVIL Portal. Do this by clicking the `Up` arrow in the Data Cruncher window and selecting it from among the files on your computer.
+    - <img src="./rna-seq-images/rna-seq-9-19.png" width="400"/>
+
+- The final paragraph includes three final variables that must be set to complete the transfer.
+    - Set `target_platform = 'cavatica'`
+    - Set `target_project = 'USER/PROJECT'`, where `USER` is your CAVATICA username and `PROJECT` is the name of the CAVATICA project you wish to move the files to. You may also copy these from the URL of the project after navigating there in your browser window.
+    - Set `pfb = 'PATH'`, where `PATH` is the path to the .avro PFB file you uploaded. You can copy the path by right-clicking the file and choosing `Copy Path`. 
+        - <img src="./rna-seq-images/rna-seq-9-20.png" width="400"/>
+- When all three variables have been set, use the `Play` button to run this paragraph. The files will begin to transfer and appear one-by-one below the paragraph.
```

Comment:
I have set all three variables, but nothing happens when I click the play button. 

```
# Select the platform you are working on
target_platform = 'cavatica' # 'cgc' for CGC, 'bdc' for BioData Catalyst, 'cavatica' for CAVATICA

# Select the project to import the data
target_project = 'rmharris8/copy-of-data-interoperability' # slug from URL: username/project_name

# Path to pfb file, e.g. ./test-data/gtex_analysis_txt.avro
pfb = 'export_2022-01-28T17_09_23.avro'

gtex_records = get_sequencing_records(pfb_file=pfb, client=interop_client)

# Note that this step can take longer for bigger files
for record in gtex_records:
    record.import_to_project(platform=target_platform, project=target_project)
```

Reply:
update, i had missed adding the 7 bridges token. now, i get an error message when I click play.

```
---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
/tmp/ipykernel_204/1619340111.py in <module>
     12 # Note that this step can take longer for bigger files
     13 for record in gtex_records:
---> 14     record.import_to_project(platform=target_platform, project=target_project)

/sbgenomics/workspace/interop.py in import_to_project(self, platform, project)
    364         )
    365         if self.anvil_client.api_key:
--> 366             file_url = self.anvil_client.import_drs(
    367                 self.seq_object.drs_import_link)
    368         else:

/sbgenomics/workspace/interop.py in import_drs(self, drs_link, access_id)
    174         drs_split = without_drs_part.split(':')
    175         drs_id = drs_split[1]
--> 176         drs_object = self.get_file_url(drs_id, access_id)
    177         return drs_object['url']
    178 

/sbgenomics/workspace/interop.py in get_file_url(self, drs_id, access_id)
    162                 drs_id, access_id, r['stef_status_code'], r['message']
    163             )
--> 164             raise Exception('Unable to get file with drs')
    165 
    166         return r

Exception: Unable to get file with drs
```